test/unit/api/test_api.py

```diff
@@ -94,7 +94,6 @@ def test_get_file_info(self):
                     'mode': 'none'
                 },
                 'uploadTimestamp': 5000,
-                'cacheControl': 'private, max-age=3600',
```

Comment:
This test expected this because the raw simulator was providing `cacheControl` (wrongly) and now it isn't.